opm/io/eclipse/EGrid.hpp

```diff
@@ -53,27 +54,35 @@ class EGrid : public EclFile
 
     void load_grid_data();
     void load_nnc_data();
+    void mapaxes_init();
+    void mapaxes_transform(double& x, double& y) const;
```

Comment:
I don't think the `init()` and `transform()` functions need to be public.  They're effectively details internal to the implementation.